src/main/java/com/minecolonies/util/LanguageHandler.java

```diff
@@ -62,8 +63,10 @@ public static String getString(String key)
      * @param defaultValue the value to return if no key is found
      * @return Localized string
      */
+    @SuppressWarnings("deprecation")
```

Comment:
Well minecraft uses Deprecation for things like "Internal, don't use" and we have to say "For this one usage, we need the internal thing.

And often minecraft says "Deprecated, but the alternative comes in the version where we remove this altogether" So no other option there. 

I like the compiler flag so we spot new deprecations and we suppress the ones we know we can't fix on our end